Exercises/SumOfMultiples/SumOfMultiples.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercises.SumOfMultiples
+{
+    internal static class SumOfMultiples
+    {
+        public static int Sum(IEnumerable<int> multiples, int max)
```

Comment:
Lendo o exercício, entendi que você teria de achar os multiplos... mas você os recebe via parâmetro. Por quê? O que está dentro dessa lista de multiples?

Reply:
Poderia usar um nome mais significativo, tipo: 

```suggestion
        public static int Sum(IEnumerable<int> multiplesBase, int max)
```